test/core/consensus/synchronizer/synchronizer_test.cpp

```diff
@@ -38,9 +41,10 @@ class SynchronizerTest : public testing::Test {
     block2_hash_.fill(4);
 
     synchronizer_ =
-        std::make_shared<SynchronizerImpl>(peer_info_, tree_, headers_);
+        std::make_shared<SynchronizerImpl>(*host_, peer_info_, tree_, headers_);
```

Comment:
I feel like synchronizer should not know about host 🤔 
synchronizer should use something that knows about host